docs/administration/Query_planning.md

```diff
@@ -193,6 +193,104 @@ By removing the specific expressions (only keep the operators), the query plan c
 
 ![8-5](../assets/8-5.png)
 
+## Query hint
+
+Query hints are directives or comments that explicitly suggest the query optimizer on how to execute a query. Hints only take effect within a single query.
+
+### System variables hint
+
+You can set one or more system variable hints by using the `/*+ ... */` comment syntax in SELECT and SUBMIT TASK statements. You can also use system variable hints in SELECT clauses that included in other statements such as CREATE MATERIALIZED VIEW AS SELECT and CREATE VIEW AS SELECT.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/91118bfe-5ddf-43d6-9e00-65cfb3dec57d)
The two sentences can be combined into one: You can set one or more system variable hints by using the `/*+ ... */` comment syntax in SELECT and SUBMIT TASK statements, or in SELECT clauses that are (NOTE that "are" should be added between "that" and "included") included in other statements such as CREATE MATERIALIZED VIEW AS SELECT and CREATE VIEW AS SELECT.